test/js/pageobjects/PoaGuidance.js

```diff
@@ -0,0 +1,68 @@
+import Base from './BasePage.js'
+import {locale, verifyElementCopy} from '../utils/mochaw'
+
+class PoaGuidance extends Base {
+  get title() { return this.$('.onfido-sdk-ui-PageTitle-titleSpan')}
+  get subtitle() { return this.$('.onfido-sdk-ui-ProofOfAddress-Guidance-subTitle')}
+  get makeSure() { return this.$('.onfido-sdk-ui-ProofOfAddress-Guidance-makeSure')}
+  get logoText() { return this.$('.onfido-sdk-ui-ProofOfAddress-Guidance-label:nth-child(7)')}
+  get fullNameText() { return this.$('.onfido-sdk-ui-ProofOfAddress-Guidance-label:nth-child(1)')}
+  get currentText() { return this.$('.onfido-sdk-ui-ProofOfAddress-Guidance-label:nth-child(2)')}
+  get addressText() { return this.$('.onfido-sdk-ui-ProofOfAddress-Guidance-label:nth-child(3)')}
+  get issueDateText() { return this.$('.onfido-sdk-ui-ProofOfAddress-Guidance-label:nth-child(4)')}
+  get summaryPeriodText() { return this.$('.onfido-sdk-ui-ProofOfAddress-Guidance-label:nth-child(6')}
+  get continueButton() { return this.$('.onfido-sdk-ui-Button-button-text')}
+
+  copy(lang) { return locale(lang) }
+
+  async verifyUIElementsForBankStatementGuidanceScreen(copy) {
```

Comment:
`verifyUIElementsForBankStatementGuidanceScreen`, `verifyUIElementsForUtilityBillGuidanceScreen`, 
`verifyUIElementsForCouncilTaxLetterGuidanceScreen`,
`verifyUIElementsForBenefitsLetterGuidanceScreen` are identical. The only thing that changes is the copy. I think you could do something like:
```
async verifyUIElementsForDocType(poaDocType, copy) {
    const poaGudanceTitleStrings = `copy.capture.${poaDocType}.front`
    const poaGudanceStrings = copy.proof_of_address.guidance
    verifyElementCopy(this.title, poaGudanceTitleStrings.title)
    verifyElementCopy(this.makeSure, poaGudanceStrings.make_sure_it_shows)
    verifyElementCopy(this.logoText, poaGudanceStrings.logo)
    verifyElementCopy(this.continueButton, poaGudanceStrings.continue)
}
```
Then you have an array with all the poa doc types and you iterate over it.
```
docTypes = [bank_building_society_statement, utility_bill, council_tax, council_tax]
async testElementsCopy() {
     docTypes.forEach( (poaDocType) => {
     verifyUIElementsForDocType(poaDocType, poa)
  })
}
```
Then you just need to call one function and there is no repetition.

Reply:
Pushed the changes in 1982664

I didn't apply the same suggestion exactly because looping through the titles of the documents is not possible with it. It requires the back navigation and selection of the different document from the PoA document selection screen for the title to change. 

I created one function though and used the suggested array. I use the array's element based on index:

`const poaGudanceTitleStrings = copy['capture'][docTypes[index]]['front']` 

instead of `const poaGudanceTitleStrings = `copy.capture.${poaDocType}.front`

Reply:
Oh, I see. For some reason I thought we were on the poa document selection screen.